Sources/PointFree/Admin/FreeEpisodes.swift

```diff
@@ -52,7 +51,7 @@ private func sendEmail(forFreeEpisode episode: Episode, toUsers users: [User]) -
 
   // A personalized email to send to each user.
   let freeEpisodeEmails = users.map { user in
-    lift(IO { inj2(freeEpisodeEmail.view((episode, user))) })
+    lift(IO { inj2(freeEpisodeEmail((episode, user))) })
```

Comment:
Email views are still pullback-composed, so the tupling prevails.

Reply:
yeahhh